specification/provisioningservices/resource-manager/Microsoft.Devices/2017-11-15/examples/DPSCreate.json

```diff
@@ -0,0 +1,70 @@
+{
+    "parameters": {
+      "provisioningServiceName": "myFirstProvisioningService",
+      "resourceGroupName": "myResourceGroup",
+      "api-version": "2017-11-15",
+      "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0",
+      "iotDpsDescription": {
+      "location": "East US",
+      "type": "Microsoft.Devices/ProvisioningServices",
+      "properties": {
+      },
+      "sku": {
+        "name": "S1",
+        "tier": "Standard",
+        "capacity": 1
+      },
+      "tags": {}
+    }
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "etag": "AAAAAAAADGk=",
+          "name": "myFirstProvisioningService",
+          "location": "eastus",
+          "properties": {
+            "state": "Active",
+            "allocationPolicy": "Hashed",
+            "serviceOperationsHostName": "myFirstProvisioningService.azure-devices-provisioning.net",
+            "deviceProvisioningHostName": "global.azure-devices-provisioning.net",
+            "idScope": "0ne00000012",
+            "authorizationPolicies": []
```

Comment:
Should have atleast one  'provisioningServiceOwner' rule.

Reply:
this is autocreated if not passed